src/scripting/macros.md

```diff
@@ -1,7 +1,8 @@
 # Macros
 
-Apart from simple sequencing and looping, rizin allows to write
```

Comment:
Add an output of `(?` here as well, I think.

Reply:
Output of `(?` goes later in this section.

Reply:
Actually is not there.

Reply:
Maybe replace this with the explaination like as follows
------------------------
Apart from simple sequencing and looping, Rizin allows to write simple macros, using this construction:

```
[0x00000000]> (?
Usage: $[*?]   # Alias commands and strings
| $[alias[=cmd] [args...]] # List all defined aliases / Define alias (see %$? for help on $variables)
| $*                     # List all the aliases as rizin commands in base64
| $**                    # Same as above, but using plain text

Detailed help for $[alias[=cmd] [args...]] is provided by $??.
```
Example of macro usage named `qwe`:
```
[0x00404800]> (qwe; pd 4; ao)
```